mkdocs/docs/experiments/categories/load/k6-loadgen.md

```diff
@@ -0,0 +1,184 @@
+## Introduction
+
+[k6](https://k6.io/) loadgen fault simulates load generation on the target hosts for a specific chaos duration. This fault:
+- Slows down or makes the target host unavailable due to heavy load.
+- Checks the performance of the application or process running on the instance.
+Support [various types](https://grafana.com/docs/k6/latest/testing-guides/test-types/) of load testing (ex. spike, smoke, stress)
+
+!!! tip "Scenario: Load generating with k6"    
+    ![k6-loadgen](../../images/k6-loadgen.png)
+
+## Uses
+
+??? info "View the uses of the experiment" 
+    coming soon
+
+## Prerequisites
+
+??? info "Verify the prerequisites" 
+    - Ensure that Kubernetes Version > 1.16 
+    - Ensure that the Litmus Chaos Operator is running by executing <code>kubectl get pods</code> in operator namespace (typically, <code>litmus</code>).If not, install from <a href="https://docs.litmuschaos.io/docs/getting-started/installation">here</a>
+    - Ensure to create a Kubernetes secret having the JS script file in the `Chaos Infrastructure`'s namespace(typically litmus). The simplest way to create a secret object looks like this:
+            ```bash
+            kubectl create secret generic k6-script \
+                --from-file=script.js=<<script-path>> -n <<chaos_infrastructure_namespace>>
+            ``` 
+
+## Default Validations
+
+??? info "View the default validations" 
+    The application pods should be in running state before and after chaos injection.
```

Comment:
We don't have any application pods here, right? can we update this validation?

Reply:
Right, I deleted this section.